### PR TITLE
Can view entries

### PR DESCRIPTION
**Pull request by addy-abid**
https://user-images.githubusercontent.com/85080021/138965530-3e36ca19-63cc-4055-83ce-ecd6668b8c1d.mp4

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work, dude! Two notes to address:
1. When a new entry is submitted, it needs to be visible as soon as the user is redirected to the entries list, it should not require that the page is refreshed.
2. New entries should sit at the top of the list of entries, not at the bottom. Entries should be shown in a list ordered from newest to oldest.

**Comment by addy-abid** [comment]
https://user-images.githubusercontent.com/85080021/138967735-ef0b5b17-d92b-4bf6-9ce2-7f00410cb13b.mp4

File: js/main.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ function formSubmit(event) {
   $entryForm.reset();
   renderImage();
   swapViews(data.view);
-
+  location.reload();
```
Comment (codingcodymiller) [inline js/main.js]: `location.reload()` _**does**_ refresh the page, this is not the way to go. You need to update the application without needing to entirely rebuild it from scratch. When a new entry is submitted, you need to generate a DOM tree for that entry, then `prepend` it to the list in the entries view. That will achieve the same effect without needing to refresh the page entirely

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by addy-abid** [comment]
https://user-images.githubusercontent.com/85080021/138972516-43d980e9-9199-4fb8-81b1-d8e9faa8fb4b.mp4

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**